pyuvdata/cst_beam.py

```diff
@@ -237,6 +255,13 @@ def read_cst_beam(self, filename, beam_type='power', feed_pol='x',
             theta_phase = theta_phase.reshape((theta_axis.size, phi_axis.size), order='F')
             phi_phase = phi_phase.reshape((theta_axis.size, phi_axis.size), order='F')
 
+            if theta_mag_dbi:
+                theta_mag = 10. ** (theta_mag / 20.)
+                theta_phase *= .5
```

Comment:
Why are you dividing the phase by 2 here?

Reply:
I'm not sure why I did that. I've removed it.